### ISSUE TITLE
EventStream element creation is not valid

### ISSUE THREAD

**Issue by eceozturk** [Dash-Industry-Forum/dash-live-source-simulator#71]
https://testassets.dashif.org/#testvector/details/58a5da727459f8cb201b8a56

This test vector has `EventStream` element with `messageData` attribute; however, `messageData` attribute is an attribute of `Event` element according to ISO/IEC 23009-1.

In mpdprocessor.py the `create_inline_mpdcallback_elem(BaseURLSegmented)` function assigns `messageData` attribute to `EventStream` element.

**Comment by TobbeEdgeware** [comment]
Yeah. You seem to be right. @waqarz can you look more into this since this was done by your team (Aug. 2016). The URL is also pointing to vm2.dashif.org, so it should be changed to livesim.dashif.org

**Comment by tobbee** [comment]
Fixed in #110
